ods-devenv/openvpn/build-client-profile.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+#######################################
+# Run this script in the EDP server
+# ./build-client-profile.sh {EC2_INSTANCE} client1 > clientProfile.ovpn
+# This script is to generate the OpenVPN client profile for the EDP in a box.
+# This profile can be used from an openvpn-connect client
+# (see https://openvpn.net/downloads/openvpn-connect-v3-windows.msi)
+# Arguments:
```

Comment:
Can you provide an "Example:" block with an example script call underneath the "Arguments:" block?